beaker/session.py

```diff
@@ -548,8 +540,15 @@ def __init__(self, request, key='beaker.session.id', timeout=None,
                 cookie_data = self.cookie[self.key].value
                 self.update(self._decrypt_data(cookie_data))
```

Comment:
This is the line that was previously silently failing because of the try/except inside _decrypt_data